scripts/Tools/Makefile

```diff
@@ -69,12 +69,12 @@ endif
 ifeq ($(strip $(USE_ALBANY)), TRUE)
    USE_CXX = true
 endif
-ifeq ($(strip $(USE_KOKKOS)), TRUE)
-   USE_CXX = true
-endif
-
-ifeq ($(strip $(USE_FMS)), TRUE)
-  SLIBS += -lfms
+ifeq ($(CIME_MODEL),e3sm)
+  HOMME_TARGET ?= $(shell $(CASEROOT)/xmlquery --caseroot $(CASEROOT) CAM_TARGET --value)
```

Comment:
Why was this change made? It looks like it adds-back HOMME checks that I removed.

Reply:
i'll remove it again - it's hard to change stuff you can't test...